### ISSUE TITLE
Problem on commit "Finished Milestone 3"

### ISSUE THREAD

**Issue by taduyvinh** [CoderClass/ruby-lab1-longteam#1]
Hello. Can you check the commit again? The changed files in the last commit still differs from what you did during milestone 3. When i pulled to my local repo it appears that the progress is still at Milestone 2. Thank you!

**Comment by vulong264** [comment]
are you sure you are pulling from the right branch? Anyway, I've just re-pushed the whole thing again. Already checked on web-browser. Everything is the same between my local and github. Try have a look at the file app/views/home/index.html.erb on Github web, the current file is as follow:

`<div class="jumbotron">
  <h1 class="display-3">CoderPhoto</h1>
  <p class="lead">This is a starter project for your pair-programming lab.</p>
  <hr class="m-y-2">
  <p>Please work together to complete as many milestones as you can.</p>
  <p class="lead">
    <a class="btn btn-danger btn-lg" href="http://learning.coderschool.vn/courses/_ruby_on_rails/unit/1#!exercises" role="button" target="_blank">Learn more</a>
    </p>
</div>
`

**Comment by taduyvinh** [comment]
Yes thank you now it's updated perfectly. Gonna commmit my part tomorrow

**Comment by vulong264** [comment]
hey, have you finished milestone 4? I haven't seen any changes yet?

**Comment by taduyvinh** [comment]
Just pushed to github, bundle to install gem Faker and see if the "Photos" page now full of random Photos.

**Comment by vulong264** [comment]
Did you push html files under photo path? I don't see any changes there yet?

-- 
Vu Long
Mobile:+84.989.099.338
Skype: mr.vulong

On October 12, 2016 at 11:27:51 AM, taduyvinh (notifications@github.com) wrote:

Just pushed to github, bundle to install gem Faker and see if the "Photos" page now full of random Photos.

**Comment by taduyvinh** [comment]
yeah i only need to modify "seed.rb" file for milestones 4. Now i'm working on the bonus so there might be changes in the html files

**Comment by vulong264** [comment]
Well, nothing showing on my photo page, just the same 2 old photo. Already install gem Faker. What might be wrong?

-- 
Vu Long
Mobile:+84.989.099.338
Skype: mr.vulong

On October 12, 2016 at 11:41:04 AM, taduyvinh (notifications@github.com) wrote:

yeah i only need to modify "seed.rb" file for milestones 4. Now i'm working on the bonus so there might be changes in the html files

**Comment by taduyvinh** [comment]
oh i think you need to run 'rails db:seed' first 
to get back to our original photos, just 'rails db:fixtures:load' again

**Comment by vulong264** [comment]
OK it works now. I think next time we should include these kind of instruction in the comment while committing new changes!

-- 
Vu Long
Mobile:+84.989.099.338
Skype: mr.vulong

On October 12, 2016 at 11:44:28 AM, taduyvinh (notifications@github.com) wrote:

oh i think you need to run 'rails db:seed' first
to get back to our original photos, just 'rails db:fixtures:load' again

**Comment by taduyvinh** [comment]
ok that's a good idea. I'll be sure to note those down

**Comment by vulong264** [comment]
Hey, I've just pushed the code to complete all additional features. Have a quick check for me will you?
The app is also up running at https://murmuring-headland-68382.herokuapp.com/

-- 
Vu Long
Mobile:+84.989.099.338
Skype: mr.vulong

On October 12, 2016 at 11:47:28 AM, taduyvinh (notifications@github.com) wrote:

ok that's a good idea. I'll be sure to note those down

**Comment by taduyvinh** [comment]
Sorry for the late checking. And wow you finished them well. All the features work well on the local repo, but on the heroku link i can only see the upload photos features. Maybe the pagination and comment features are updated but there's no fake record so they don't appear on the site?

**Comment by vulong264** [comment]
Yeah, by default heroku doesn't support Faker on production environment. It
takes time for a work around so I just leave it there for now :D

**Comment by taduyvinh** [comment]
ok let's continue working on our assignment
